README.md

```diff
@@ -143,15 +147,11 @@ contained changes as well as the ability to diff agains the previous version.
 - There is a workaround to removing the PID lock of the Rails process in the
   event of the application crashing and not releasing the process.
 - The Rails secret is created here.
-- We have to pass the `APPLICATION_ROOT` and, depending on the ivocation choice
-  also pass in the `API_SERVICE_URL`, so that they are available to the
-  `entrypoint.sh` otherwise the application will throw an error and exit before
-  starting.
 
-#### Configuration environment variables
+### Configuration environment variables
 
-We use a number of environment variables to determine the runtime behaviour of
-the application:
+We can use a number of environment variables to determine the runtime behaviour
+of the application while developing the codebase locally:
```

Comment:
Env var table needs re-write.

Reply:
I'm working on unpicking the comments and threads found on https://github.com/epimorphics/hmlr-linked-data/issues/52 in order to resolve https://github.com/epimorphics/hmlr-linked-data/issues/79 and your comment.